tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_time_check.py

```diff
@@ -0,0 +1,45 @@
+import time
+
+from marionette.wait import Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.lockscreen.app import LockScreen
+
+
+class TestLockScreen(GaiaTestCase):
+
+    _seconds_since_epoch = 1357043430  # the time is 12:30 GMT
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        #self.connect_to_local_area_network()
+
+    def test_lockscreen_time_check(self):
+
+        # disable time/timezone automatic update
+        self.data_layer.set_setting('time.clock.automatic-update.enabled', False)
+        self.data_layer.set_setting('time.timezone.automatic-update.enabled', False)
+
+        # set time manually to 12:30
+        self.data_layer.set_time(self._seconds_since_epoch * 1000)
+        self.data_layer.set_setting('time.timezone', 'Atlantic/Reykjavik')
```

Comment:
It seems more prudent to me to change the timezone before the time because it might lead to intermittent failures. Let's say we're in PST, we change the time to 12:30PM and the change is immediately taken into account. Then we change it to UTC => It'll be 8:30PM.